[BUG] panic when `kubekit describe cluster` is executed

<!--
A great way to contribute to the project is to send a detailed report when you
encounter an issue. We always appreciate a well-written, thorough bug report,
and will thank you for it!
The GitHub issue tracker is for bug reports and feature requests. General support can be found at Slack - kubekit-workspace.slack.com #development channel
-->

**Community Note**

* Please, make sure that we do not have any duplicate issues already open. Search the issue list for this
repository and if there is a duplicate, please close your issue and add a comment to the existing issue instead, you can use the "subscribe" button to get notified on updates
* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* If you have ways to reproduce the issue or have additional information that may help
resolving the issue, please leave a comment.

**Describe the bug**
A `panic: runtime error: invalid memory address or nil pointer dereference` error happen when try to get a cluster description with `kubekit describe cluster NAME`

**To Reproduce**
Execute `kubekit describe cluster CLUSTER-NAME`

**Expected behavior**
No panic

**Screenshots or logs**

```bash
kubekit describe cluster pr243eks
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x7c3678d]goroutine 1 [running]:
github.td.teradata.com/kubekit/kubekit/cli/kubekit.describeClusterRun(0x1005a7c0, 0xc0008ad540, 0x1, 0x1, 0x0, 0x0)
/workspace/kubekit/kubekit/cli/kubekit/describe.go:74 +0x15d
```

**Settings**
Defaults


**Versions:**

```bash
kubekit version --verbose
KubeKit v2.1.0+build.0.397ec957
Kubernetes version: v1.15.6
```

**Environment variables:**
Defaults

--- a/cli/kubekit/describe.go
+++ b/cli/kubekit/describe.go
@@ -59,10 +59,14 @@ func addDescribeCmd() {
 	describeCmd.Flags().String("format", "", "pretty-print the cluster configuration using a Go template")
 
 	describeCmd.AddCommand(describeClusterCmd)
-	// describe templates NAME[,NAME ...] --output (json|yaml|toml) --pp
+	// describe templates NAME[,NAME ...] --output (json|yaml|toml) --pp --format ...
+	describeClusterCmd.Flags().String("format", "", "pretty-print the cluster configuration using a Go template")
+
 	describeCmd.AddCommand(describeTemplatesCmd)
 	// describe nodes CLUSTER-NAME --output (json|yaml|toml) --pp
+
 	describeCmd.AddCommand(describeNodesCmd)
+	// describe templates NAME[,NAME ...] --output (json|yaml|toml) --pp --cluster NAME
 	describeNodesCmd.Flags().StringP("cluster", "c", "", "cluster name where this node is located")
 }
 

